frontend/src/features/admin-form/settings/components/AuthSettingsSection/EsrvcIdBox.tsx

```diff
@@ -60,16 +61,24 @@ export const EsrvcIdBox = ({
     return errors.esrvcId ? reset() : onSubmit()
   }, [errors, onSubmit, reset])
 
-  const renderedHelperText = useMemo(() => {
+  const renderHelperTextComponent = () => {
     switch (settings.authType) {
       case FormAuthType.SP:
       case FormAuthType.CP:
       case FormAuthType.MyInfo:
-        return 'Contact askNDI@tech.gov.sg for your e-service ID'
+        return (
+          <Text textStyle="body-2" color="secondary.400">
+            Contact{' '}
+            <Link isExternal href="https://go.gov.sg/formsg-singpass-contact">
```

Comment:
Can consider moving this over to [shared/constants/links](https://github.com/opengovsg/FormSG/blob/develop/shared/constants/links) for easier lookups for external links -- I do see a couple of string constants littered around the codebase, but if possible, we want to avoid adding more to the litter 🙏

Reply:
resolved! thanks for pointing out the existence of ths file, didnt even realise it existed haha